autogluon/training/buildspec.yml

```diff
@@ -38,7 +38,7 @@ images:
   BuildAutogluonGPUTrainPy3DockerImage:
     <<: *TRAINING_REPOSITORY
     build: &AUTOGLUON_GPU_TRAINING_PY3 false
-    image_size_baseline: 16426
```

Comment:
The size increase is significant, do we know why?

Reply:
ditto and also because we install `pytorch-2.1` manually.

Reply:
same as the comment mentioned above for inference images